qldbdriver/transaction_test.go

```diff
@@ -318,6 +318,11 @@ func TestTransactionExecutor(t *testing.T) {
 		abort := testExecutor.Abort()
 		assert.Error(t, abort)
 	})
+
+	t.Run("Transaction ID", func(t *testing.T) {
+		id := testExecutor.ID()
+		assert.NotEmpty(t, id)
```

Comment:
We can assert this to [mockID](https://github.com/awslabs/amazon-qldb-driver-go/blob/1a6d63b6f4461342f55fab40c8ae3e33e68b0ef4/qldbdriver/transaction_test.go#L151)

Reply:
Addressed.